respec-config.js

```diff
@@ -22,7 +22,18 @@ var respecConfig = {
 		company: "IBM",
 		companyURI: "https://www.ibm.com/",
 		w3cid: 46880
-	}],
+	},{
+		Name: "Chris Loiselle",
+		company: "Oracle Corporation",
+		companyURI: "http://www.oracle.com/",
+		w3cid: "65246"
```

Comment:
I'm not sure whether Chris or Phil has the w3cid of 65246, but both of them are marked here with the same ID. This needs correction before merging this pull request.

Reply:
Another thing, is on my name the w3cid does not have quotes around the value, but the added ones for Phil and Chris do have quotes around the value. My guess is that quotes are not desired.